src/sentry/processing/realtime_metrics/redis.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+
+from sentry.exceptions import InvalidConfiguration
+from sentry.utils import redis
+
+from . import base
+
+
+class RedisRealtimeMetricsStore(base.RealtimeMetricsStore):
+    def __init__(
+        self,
+        cluster: str,
+        counter_bucket_size: int,
+        counter_ttl: datetime.timedelta,
+        prefix: str,
```

Comment:
we no longer have a default prefix set and it is also not set in the config!  that's a problem i think.

to be fair i think we should just hardcode the prefix for now.

Reply:
but this should have been caught by tests.  I think we should add a test that tries to use the default by using `sentry.processing.realtime_metrics.increment_project_event_counter()` directly and see if it works.

Reply:
I meant to add it to the config, but forgot. I'll just hard code it for now.